wandb/integration/lightgbm/__init__.py

```diff
@@ -94,12 +94,15 @@ def _log_feature_importance(model: "Booster") -> None:
     )
 
 
-def wandb_callback(log_params: bool = True, define_metric: bool = True) -> Callable:
+def wandb_callback(
+    log_params: bool = True, define_metric: bool = True, sep: str = "/"
```

Comment:
My only question is should we keep backward computability here? i.e. sep="_" and add in the documentation comment that changing it to "/" will create the hierarchical view? 

I don't have strong feelings here and open to hear what others think